libs/feature/editor/src/lib/components/generic-keywords/generic-keywords.component.html

```diff
@@ -4,9 +4,10 @@
     [displayWithFn]="displayWithFn"
     [action]="autoCompleteAction"
     (itemSelected)="handleItemSelection($event)"
-    [preventCompleteOnSelection]="true"
-    [minCharacterCount]="0"
+    [preventCompleteOnSelection]="false"
```

Comment:
I see `false` is the default value, but it might indeed be more explicit to set it here.